### ISSUE TITLE
🔗 🧩 🐞 Bug Report: Elementor/Popup - Crocoblock Filters 3.9 (Related to #20708)

### ISSUE THREAD

**Issue by rally-lad** [elementor/elementor#20713]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Upgraded from 3.8.1 / 3.8.2 Pro to 3.9

Elementor Popup is used as a side bar to host Crocoblock filters.

Worked previously, but now the filters no longer work within the popup in 3.9.

If I drag / paste the filters to the main Elementor archive product page they then work.

### Steps to reproduce

Create popup with crocoblock product filter and link to button on woocommerce archive page 
Products are no longer filtered in the latest version of Elementor

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.37
	PHP Version: 8.1.12
	PHP Memory Limit: 256M
	PHP Max Input Vars: 2000
	PHP Max Post Size: 20M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: *
	Home URL: *
	WP Multisite: No
	Max Upload Size: 20 MB
	Memory limit: 1G
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: *
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	Disable Bloat for WordPress & WooCommerce
		Version: 3.1.0
		Author: Disable Bloat

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Elementor
		Version: 3.9.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	JetSmartFilters
		Version: 3.0.2
		Author: Crocoblock

	OoohBoi Steroids for Elementor
		Version: 2.0.5
		Author: OoohBoi

	Sequential Order Numbers for WooCommerce
		Version: 1.9.7
		Author: SkyVerge

	Store Manager Connector
		Version: 1.1.2
		Author: eMagicOne

	UpdraftPlus - Backup/Restore
		Version: 1.22.23
		Author: UpdraftPlus.Com, DavidAnderson

	Visibility Logic for Elementor
		Version: 2.3.2
		Author: StaxWP

	WooCommerce
		Version: 7.1.1
		Author: Automattic

	WooCommerce Brands
		Version: 1.6.38
		Author: WooCommerce

	WooCommerce Stripe Gateway
		Version: 7.0.1
		Author: WooCommerce

	WP Mail SMTP
		Version: 3.6.1
		Author: WPForms

	Yoast SEO
		Version: 19.11
		Author: Team Yoast

== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved asset loading: Active
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive
	Accessibility improvements: Active
	Additional Custom Breakpoints: Active
	Import Export Template Kit: Inactive
	Hide native WordPress widgets from search results: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Landing pages: Inactive
	Colour Sampler: Inactive
	Favourite Widgets: Inactive
	Admin Top Bar: Inactive
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Inactive
	Loop: Inactive by default
	Form Submissions: Inactive
	Scroll Snap: Inactive

== Log ==
	

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	OoohBoi Steroids for Elementor: Compatibility not specified
	Visibility Logic for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by 79ho3ein** [comment]
Hi @rally-lad
This is not a problem on Elementor's part. I discussed this with the developer of jet smart filters and he said an update for it will be released this week.

**Comment by mnemotiv** [comment]
temporary workaround (backwards compatibility):

```js
window.addEventListener( 'elementor/popup/show', function ( e ) {
	$( document ).trigger( 'elementor/popup/show', [ e.detail.id, e.detail.instance ] );
} );

window.addEventListener( 'elementor/popup/hide', function ( e ) {
	$( document ).trigger( 'elementor/popup/hide', [ e.detail.id, e.detail.instance ] );
} );
```

of course, there's "Elementor way" to work with events, but...

**Comment by nicholaszein** [comment]
Hello @rally-lad,

Thank you for reaching out! 🙏 

### ⛔ Please note that while we do welcome bug reports, we can't provide support for 3rd party add-ons.

If your issue is specifically happening only when you're using a feature from a 3rd party add-on, or only when the 3rd party add-on is active, you'll need to contact the support for that add-on or its developers.

Regards

**Comment by nicholaszein** [comment]
Hello @rally-lad

Apologies for closing the issue too rapidly. 🙏 

I see now this issue is related #20708.

We have a fix underway and we will update you when it's fixed.

Kind regards

**Comment by hitsgee** [comment]
I believe Crocoblock released a patch for this in the latest filters update.

**Comment by rami-elementor** [comment]
Fixed in Elementor Pro 3.9.2
